### PR TITLE
(#271) Documentation added for module: port_http_server

### PR DESCRIPTION
**Pull request by sumitkp18**
## Added documentation for classes of port_http_server module

---

For the Pull Request to be accepted please check:

- [x] The PR has a meaningful title.
- [x] If the PR refers to an issue, it should be referenced with the Github format (*#ID*).
- [x] The PR is done to the `develop` branch.
- [x] The code pass all PR checks.
- [ ] All public members are documented using [Dokka](https://github.com/Kotlin/dokka).
- [ ] The code follow the coding conventions stated at the [contributing.md](/contributing.md) file.

### CODE REVIEW

File: port_http_server/src/main/kotlin/Call.kt [hunk 1]
```diff
@@ -39,14 +39,34 @@ class Call(val request: Request, val response: Response, val session: Session) {
...
+     *
+     * @param content content of the response
+     * @param contentType content type of the resposne
+     */
     fun ok(content: Any = "", contentType: String? = null) = send(200, content, contentType)
 
+    /**
+     * Sends success response serialized using given [SerializationFormat] and [Charset].
+     *
+     * @param content content of the response
```
Comment (jaguililla) [inline port_http_server/src/main/kotlin/Call.kt:53]: For proper documentation formatting, comments should be capitalized and end with a point. I.e.:

    @param content Content of the response.

Would you mind changing that? Thank you very much!

File: port_http_server/src/main/kotlin/Request.kt [hunk 2]
```diff
@@ -22,42 +22,169 @@ import kotlin.reflect.KClass
...
+     */
     val acceptValues: List<String> by lazy { headersValues["Accept"] ?: emptyList() }
+
+    /**
+     * First value in "Accept" header key.
+     */
     val accept: String? by lazy { acceptValues.firstOrNull() }
+
+    /**
+     *
```
Comment (jaguililla) [inline port_http_server/src/main/kotlin/Request.kt:116]: You missed this one ;)

**Comment by jaguililla** [comment]
Sorry for the late reply, and thanks for documenting this module! I've added a few comments and aside of that it is good to go for me :)

File: port_http_server/src/main/kotlin/Request.kt (hunk 2 above)
Comment (sumitkp18) [reply port_http_server/src/main/kotlin/Request.kt]: Comment addressed.
Thanks.

File: port_http_server/src/main/kotlin/Call.kt (hunk 1 above)
Comment (sumitkp18) [reply port_http_server/src/main/kotlin/Call.kt]: Comment addressed at all places.
Thanks.

**Comment by jaguililla** [comment]
Good job! I'll merge it right now

**Pull request merged**